app/ui/browser/reducers/pages.js

```diff
@@ -55,19 +52,6 @@ export default function basic(state = initialState, action) {
     case types.IPC_COMMAND_CLOSE_TAB:
       return closeTab(state, action.pageId);
 
-    case types.NAVIGATE_PAGE_BACK:
-      return navigatePageBack(state, action.pageId);
-
-    case types.NAVIGATE_PAGE_FORWARD:
-      return navigatePageForward(state, action.pageId);
-
-    case types.NAVIGATE_PAGE_REFRESH:
-    case types.IPC_COMMAND_PAGE_REFRESH:
```

Comment:
In some way, we need to be able to trigger a refresh from the main process.  Perhaps you re-instate this later, or perhaps that's follow-up.